src/Expander/DeclScope.hs

```diff
@@ -2,6 +2,13 @@ module Expander.DeclScope where
 
 import Data.Unique
 
+-- | A 'DeclValidityPtr' gets filled with a 'ScopeSet' representing an
+-- environment. Each 'Scope' in this 'ScopeSet' corresponds to an identifier
+-- which is bound in that environment, and the 'ScopeSet' indicates the phase
+-- or phases in which each 'Scope' is valid.
+--
+-- Note that 'DeclValidityPtr' gets filled once we know which _names_ get
+-- bound, the values to which they are bound may not be fully-expanded yet.
```

Comment:
I changed this comment again, to reflect the fact that `DeclValidityPtr`s now point to `ScopeSet`s instead of `PhaseSpec`s.

Reply:
I changed this comment yet again, because I had used the word "environment" incorrectly.